app/helpers/catalog_helper.rb

```diff
@@ -55,9 +55,163 @@ def service_catalog_summary(record, sb_data)
     miq_structured_list(data)
   end
 
+  def catalog_basic_information(record, sb_params, provisioning, tenants_tree)
+    data = {:title => _('Basic Information'), :mode => "miq_catalog_basic_information"}
+    rows = []
+    rows.push(row_data(_('Name / Description'), "#{record.name} / #{record.description}"))
+    rows.push(row_data(_('Display in Catalog'), {:input => "checkbox", :name => "display", :checked => record.display, :disabled => true, :label => ''}))
+    rows.push(row_data(_('Catalog'), h(record.service_template_catalog ? record.service_template_catalog.name : _('Unassigned'))))
+    rows.push(row_data(_('Zone'), record.zone ? record.zone.name : '')) unless record.composite?
+    rows.push(row_data(_('Dialog'), h(sb_params[:dialog_label]))) unless catalog_provision?(record, :playbook)
+    rows.push(row_data("%s (%s #{record.currency.code})" % [_('Price / Month'), _('in')], record.price)) if record.currency
```

Comment:
Not for this PR, but this one is hard for translators because that `in` is just hanging by itself.  Better to put it all in one string, something like
```suggestion
    rows.push(row_data(_("Price / Month (in %{currency})") % {:currency => record.currency}) if record.currency
```
cc @MelsHyrule